sdk/keyvault/Azure.Security.KeyVault.Keys/samples/Sample4_EncryptDecrypt.md

```diff
@@ -0,0 +1,68 @@
+# Encrypting and decrypt keys
+
+This sample demonstrates how to encrypt and decrypt a single block of plain text with an RSA key.
+To get started, you'll need a URI to an Azure Key Vault. See the [README](../README.md) for links and instructions.
+
+## Creating a KeyClient
+
+To create a new `KeyClient` to create, get, update, or delete keys, you need the endpoint to a Key Vault and credentials.
+You can use the `DefaultAzureCredential` to try a number of common authentication methods optimized for both running as a service and development.
```

Comment:
This statement is repeated - put in an include?

Reply:
Won't work in this context.